server/src/test/scala/za/co/absa/atum/server/api/database/flows/functions/GetFlowCheckpointsIntegrationTests.scala

```diff
@@ -17,39 +17,41 @@
 package za.co.absa.atum.server.api.database.flows.functions
 
 import za.co.absa.atum.server.ConfigProviderTest
-import za.co.absa.atum.model.dto.{CheckpointQueryDTO, PartitionDTO, PartitioningDTO}
 import za.co.absa.atum.server.api.TestTransactorProvider
 import za.co.absa.atum.server.api.database.PostgresDatabaseProvider
 import za.co.absa.db.fadb.exceptions.DataNotFoundException
 import za.co.absa.db.fadb.status.FunctionStatus
-import zio.interop.catz.asyncInstance
-import zio.{Scope, ZIO}
+import zio._
 import zio.test._
+import zio.interop.catz.asyncInstance
 
 object GetFlowCheckpointsIntegrationTests extends ConfigProviderTest {
 
   override def spec: Spec[TestEnvironment with Scope, Any] = {
 
-    val partitioningDTO1: PartitioningDTO = Seq(
-      PartitionDTO("stringA", "stringA"),
-      PartitionDTO("stringB", "stringB")
-    )
-
     suite("GetFlowCheckpointsIntegrationTests")(
-      test("Returns expected sequence of flow of Checkpoints with existing partitioning") {
-        val partitioningQueryDTO: CheckpointQueryDTO = CheckpointQueryDTO(
-          partitioning = partitioningDTO1,
-          limit = Some(10),
-          checkpointName = Some("checkpointName")
+      test("Should return checkpoints with the correct flowId, limit, and offset") {
+        // Define the input arguments
+        val flowId: Long = 1L
```

Comment:
bit unnecessary to define these vals if you use them only once

Reply:
Yeah, very true.